backend/app/views/spree/admin/store_credits/index.html.erb

```diff
@@ -0,0 +1,44 @@
+<% content_for :page_title do %>
+  <%= link_to "#{@user.email}", edit_admin_user_url(@user) %> / <%= Spree.t(:"admin.user.store_credits") %>
+<% end %>
+
+<%= render partial: 'spree/admin/users/sidebar', locals: { current: :store_credits } %>
+
+<% content_for :page_actions do %>
+  <%= button_link_to Spree.t(:back_to_users_list), spree.admin_users_path, class: 'btn-primary', icon: 'chevron-left' %>
+  <%= button_link_to Spree.t(:add_store_credit), new_admin_user_store_credit_path(@user), class: "btn-success", icon: 'add' %>
```

Comment:
Every route should have spree. before it otherwise there are issues for people not mounting spree to root. So `spree.new_admin_user_store_credit_path(@user)`